static/js/robot.js

```diff
@@ -426,8 +426,7 @@ class RobotClass {
             X: this.X,
             Y: this.Y,
             dir: this.dir,
-            width: this.width,
-            height: this.height
+            type: this.robotType
```

Comment:
I'm thinking about whether this key should be `robotType` rather than just `type`, since the rest of the keys match up with the RobotClass attribute name (i.e. X->this.X, Y->this.Y, dir->this.dir) as if when we access this dictionary, we're accessing the RobotClass instance directly. Alternatively, we could change line 66 to `this.type = robotInfo.robotType;` (and the corresponding next few lines).